diff --git a/node_modules/@nebular/theme/fesm2022/nebular-theme.mjs b/node_modules/@nebular/theme/fesm2022/nebular-theme.mjs
index 33bcbc2..8bca689 100644
--- a/node_modules/@nebular/theme/fesm2022/nebular-theme.mjs
+++ b/node_modules/@nebular/theme/fesm2022/nebular-theme.mjs
@@ -19,7 +19,8 @@ import { trigger, state, transition, style, animate } from '@angular/animations'
 import { Directionality, BidiModule } from '@angular/cdk/bidi';
 import { CdkCellDef, CdkHeaderCellDef, CdkFooterCellDef, CdkColumnDef, CdkHeaderCell, CdkFooterCell, CdkCell, DataSource, DataRowOutlet, HeaderRowOutlet, FooterRowOutlet, NoDataRowOutlet, CdkCellOutlet, CdkHeaderRowDef, CdkFooterRowDef, CdkRowDef, CdkHeaderRow, CdkFooterRow, CdkRow, STICKY_POSITIONING_LISTENER, CdkTable, CdkTableModule, CDK_TABLE } from '@angular/cdk/table';
 import { coerceBooleanProperty } from '@angular/cdk/coercion';
-import { _VIEW_REPEATER_STRATEGY, _DisposeViewRepeaterStrategy } from '@angular/cdk/collections';
+const _VIEW_REPEATER_STRATEGY = 'mock_VIEW_REPEATER_STRATEGY';
+class _DisposeViewRepeaterStrategy {}
 import { ESCAPE, ENTER, SPACE, BACKSPACE, DELETE } from '@angular/cdk/keycodes';
 
 /**
